sandbox-backend/src/main/java/com/exadel/sandbox/team5/entity/Employee.java

```diff
@@ -31,11 +30,20 @@
     @Column(name = "locationId")
     private Long locationId;
 
-    @ManyToMany(cascade = { CascadeType.ALL })
-    @JoinTable(
-            name = "employee_tag",
-            joinColumns = { @JoinColumn(name = "employeeId") },
-            inverseJoinColumns = { @JoinColumn(name = "discountId") }
-    )
-    private List<Discount> discounts;
+//    @ManyToMany(cascade = { CascadeType.ALL })
+//    @JoinTable(
+//            name = "employee_discount",
+//            joinColumns = { @JoinColumn(name = "employeeId") },
+//            inverseJoinColumns = { @JoinColumn(name = "discountId") }
+//    )
+//    private List<Discount> discounts;
```

Comment:
Let's remove these line because of we will use collection of Employee Discount objects

Reply:
Ok. It was in code. I didn't touch it too.